src/WebJobs.Script.Grpc/Channel/GrpcCapabilities.cs

```diff
@@ -33,7 +34,7 @@ public void UpdateCapabilities(IDictionary<string, string> capabilities, GrpcCap
                 return;
             }
 
-            _logger.LogDebug($"Updating capabilities using {strategy} strategy. Current: {_capabilities} Incoming: {capabilities}");
+            _logger.LogDebug("Updating capabilities using {strategy} strategy. Current: {_capabilities} Incoming: {capabilities}", strategy, JsonSerializer.Serialize(_capabilities), JsonSerializer.Serialize(capabilities));
```

Comment:
Question: Do we care about logging these capability entries? We are calling `Serialize` twice here, which is in the cold start code path (env reload request)

Reply:
I can definitely see this being helpful for investigations and we also use this to see how many people are using the aspnet integration feature (by checking for the capability)

cc: @brettsam

Reply:
this is okay -- there's lots of logging we do throughout the specialization flow and it's valuable. Plus, that `capabilities.ToString()` was actually serializing behind-the-scenes (`MapField<>.ToString()` serializes to json)